base/exports.jl

```diff
@@ -1233,6 +1233,7 @@ export
     @html_str,
     @doc,
     @doc_str,
+    docstrloc,
```

Comment:
I was on the fence about whether to export it from `Base` or not.  I think we envisioned `docstrloc` as a function people would use directly to locate docstrings, to make editing them a bit easier.